build.sbt

```diff
@@ -223,6 +223,32 @@ lazy val client = (project in file("target/clients/java"))
     }
   )
 
+lazy val pythonClient = (project in file("target/clients/python"))
+  .enablePlugins(OpenApiGeneratorPlugin)
+  .settings(
+    name := s"$artifactNamePrefix-client",
+    commonSettings,
+    (Compile / compile) := ((Compile / compile) dependsOn generate).value,
+
+    // OpenAPI generation specs
+    openApiInputSpec := (file(".") / "api" / "all.yaml").toString,
+    openApiGeneratorName := "python",
+    openApiOutputDir := (file("target") / "clients" / "python").toString,
+    openApiPackageName := s"$artifactNamePrefix",
+    openApiAdditionalProperties := Map(
```

Comment:
are there any more additional properties that we can set so that the generated client is more ready to be published to PyPi? perhaps to make the setup.py more PyPi ready?

Reply:
also happy to hear if there is a better way to write setup.py

Reply:
I couldn't find any openapi generator options to update the other `setup.py` values. I guess this is just a limitation of the tool. I imagine that a contributor would either manually do the various steps, including updating setup.py, before publishing, or we could have a github action that does it, which can be a follow-up PR.